src/main/java/seedu/address/MainApp.java

```diff
@@ -80,10 +80,13 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         try {
             campusConnectOptional = storage.readCampusConnect();
             if (!campusConnectOptional.isPresent()) {
-                logger.info("Creating a new data file " + storage.getCampusConnectFilePath()
-                        + " populated with a sample CampusConnect.");
+                // logger.info("Creating a new data file " + storage.getCampusConnectFilePath()
```

Comment:
It will be better to remove extra comment lines.

Reply:
Resolved in commit 26c133fe3b549dc119e905ad649fed1c16a1c0fa